WPMedia/fusion-news-theme-blocks
# TMEDIA-198 - Top table list extract options

## Description
Update top table list to make use of shared components.

## Jira Ticket
- [TMEDIA-198](https://arcpublishing.atlassian.net/browse/TMEDIA-198)

## Test Steps

1. Checkout this branch `git checkout TMEDIA-198-top-table-list-extract-options`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/top-table-list-block,@wpmedia/shared-styles`
3. Verify top table list works as before
    * Example page - http://localhost/pf/overline-top-table/?_website=the-gazette
    * http://localhost/pf/top-table-list-configs/?_website=the-gazette

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
